tutorials/multi-tenant-tutorial.rst

```diff
@@ -0,0 +1,184 @@
+    
+::
+                                                                                         
+    ALTER TABLE companies ADD PRIMARY KEY (id);                                                  
+    ALTER TABLE campaigns ADD PRIMARY KEY (id, company_id);                                      
+    ALTER TABLE ads ADD PRIMARY KEY (id, company_id);
+
+Distributing tables and loading data
+-------------------------------------
+
+We will now go ahead and tell Citus to distribute these tables across the different nodes we have in the cluster. To do so,
+you can run :code:`create_distributed_table` and specify the table you want to shard and the column you want to shard on.
+In this case, we will shard all the tables on the :code:`company_id`.                             
+                                                                                          
+::
+    
+    SELECT create_distributed_table('companies', 'id');                                       
+    SELECT create_distributed_table('campaigns', 'company_id');                               
+    SELECT create_distributed_table('ads', 'company_id');                                     
+                                                                                          
+Sharding all tables on the company identifier allows Citus to :ref:`colocate <colocation>` all the tables 
+and allow for features like primary keys, foreign keys and complex joins across your cluster.
+You can learn more about the benefits of this approach `here <https://www.citusdata.com/blog/2016/10/03/designing-your-saas-database-for-high-scalability/>`_.
+                                                                                          
+We could now go ahead and load data into these tables. For that, you can download sample data files for all the tables here:
+`companies <https://examples.citusdata.com/tutorial/companies.csv>`_, `campaigns <https://examples.citusdata.com/tutorial/campaigns.csv>`_ and `ads <https://examples.citusdata.com/tutorial/ads.csv>`_.
+
+
+.. note::
+
+    Docker users can then copy the files into the docker container in order to load them.
+    You can do so by running the docker cp command and then connecting back into the server.
+    
+    .. code-block:: bash
+           
+            docker cp PATH_TO_FILE/companies.csv citus_master:.
+            docker cp PATH_TO_FILE/campaigns.csv citus_master:.
+            docker cp PATH_TO_FILE/ads.csv citus_master:.
+            
+            docker exec -it citus_master psql -U postgres
+
+Once you have these files, you can go ahead and load them using the standard PostgreSQL :code:`\COPY` command. Please make sure that you specify the correct file path if you downloaded the file to some other location.
+
+::
+                                                                                          
+    \copy companies from 'companies.csv' WITH CSV;                                                     
```

Comment:
:-1: for this. I'd actually say lowercase `with` and `csv`. One of the things that drove me crazy about Citus in the early days was how it consistently capitalized `\STAGE`. PostgreSQL's documentation consistently lowercases client-side `psql` commands and uppercases SQL keywords. By writing `copy`, `from`, and `with csv` as lowercase, it makes it more obvious to the reader they're `psql` commands.